pkg/engine/engine.go

```diff
@@ -280,6 +281,12 @@ func NewEngine(ctx context.Context, cfg *Config) (*Engine, error) {
 
 		_, ok = results["filtered_unverified"]
 		engine.logFilteredUnverified = ok
+
+		// If retain_false_positives is set, we also need to notify unverified results.
```

Comment:
Do we _need_ to? I can buy an argument that we _should_ but I want to ensure I understand the implementation correctly.

Reply:
I just noticed if we don't my tests fail due to [this logic](https://github.com/trufflesecurity/trufflehog/blob/2964b3b2d2edf2b60b1f71443338c6534720b67a/pkg/engine/engine.go#L1101-L1104)